### PR TITLE
Fixed retry logic and simplified defaults for CL2 Addon

### PR DESCRIPTION
**Pull request by njtran**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: eksconfig/cluster-loader.go [hunk 1]
```diff
@@ -28,32 +33,8 @@ func (spec *ClusterLoaderSpec) Default(cfg *Config) {
 	}
 	if len(spec.TestConfigUris) == 0 {
 		spec.TestConfigUris = []string{
```
Comment (ellistarn) [inline eksconfig/cluster-loader.go:35]: Does this default make sense?

**Review by ellistarn — COMMENTED** [review]
Nice! You mentioned that a gotemplate change was necessary?

File: eksconfig/cluster-loader.go (hunk 1 above)
Comment (njtran) [reply eksconfig/cluster-loader.go:35]: Default makes sense to me. It's a very basic test.

**Comment by njtran** [comment]
> Nice! You mentioned that a gotemplate change was necessary?

Found that the changes you made were good enough! No change in the gotemplate needed.

**Review by ellistarn — APPROVED** [review]

**Review by ellistarn — APPROVED** [review]

**Pull request merged**